frontend/packages/console-shared/src/selectors/storage.ts

```diff
@@ -0,0 +1,4 @@
+import { K8sResourceKind } from '@console/internal/module/k8s';
+
+export const getPvcStorageSize = (pvc: K8sResourceKind): string =>
+  pvc.spec.resources.requests.storage;
```

Comment:
This is a good place for optional chaining.

```suggestion
  pvc?.spec?.resources?.requests?.storage;
```

Reply:
ack